### PR TITLE
CORTX-30891 : Setup different job for CSM tests with error code & msg check enabled

### PR DESCRIPTION
**Pull request by RajeshPratapSInghChouhan**
# Problem Statement
- Setup different job for CSM tests with error code & msg check enabled

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [x] New/Affected tests are executed on Latest Build
-  [x] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: config/__init__.py [hunk 1]
```diff
@@ -85,6 +85,8 @@ def split_args(sys_cmd: List):
     _use_ssl = '-s' if '-s' in pytest_args else '--use_ssl' if '--use_ssl' in pytest_args else None
     use_ssl = pytest_args[pytest_args.index(_use_ssl) + 1] if _use_ssl else True
     os.environ["USE_SSL"] = str(use_ssl)
+    _csm_checks = '-csm' if '-csm' in pytest_args else '--csm_checks' if '--csm_checks' in pytest_args else None
```
Comment (seagate-sarang-sawant) [inline config/__init__.py:88]: Convert this into normal if statement. This line looks unreadable and extending beyond 100 chars.

Comment (RajeshPratapSInghChouhan) [reply config/__init__.py]: updated

**Review by kachhwahadivya — APPROVED** [review]

**Comment by RajeshPratapSInghChouhan** [comment]
Test Execution Jira : 
with csm_checks True [TEST-35139](https://jts.seagate.com/browse/TEST-35139)
with csm_checks False [TEST-40804](https://jts.seagate.com/browse/TEST-40804)
without csm_checks [TEST-40758](https://jts.seagate.com/browse/TEST-40758)

Jobs:
https://eos-jenkins.colo.seagate.com/view/QA-R2/job/QA/job/QA_R2_Single_Runner_CSM_error_msg_check/13
https://eos-jenkins.colo.seagate.com/view/QA-R2/job/QA/job/QA_R2_Single_Runner_CSM_error_msg_check/14/

CLI 
python3 testrunner.py -te TEST-35139 -tp TEST-26481 -tt TODO --force_serial_run True --db_update False --target raja_prov2 --validate_certs False --use_ssl False --csm_checks True

python3 testrunner.py -te TEST-40804 -tp TEST-26481 -tt TODO --force_serial_run True --db_update False --target raja_prov2 --validate_certs False --use_ssl False --csm_checks False

python3 testrunner.py -te TEST-40758 -tp TEST-26481 -tt TODO --force_serial_run True --db_update False --target raja_prov2 --validate_certs False --use_ssl False

**Comment by RajeshPratapSInghChouhan** [comment]
[root@ssc-vm-g2-rhev4-1909 cortx-test]# pytest --collect-only --local True
========= 2118 tests collected in 5.48s =========

**Review by ketanarlulkar — APPROVED** [review]

**Pull request merged**